lib/FrontendTool/TBD.cpp

```diff
@@ -56,6 +56,27 @@ bool swift::writeTBD(ModuleDecl *M, StringRef OutputFilename,
   return false;
 }
 
+/// Determine if a symbol name is ignored when validating the TBD's contents
+/// against the IR's.
+///
+/// \param name The name of the symbol in question.
+/// \param IRModule The module being validated.
+///
+/// \returns Whether or not the presence or absence of the symbol named \a name
+///   should be ignored (instead of potentially producing a diagnostic.)
+static bool isSymbolIgnored(const StringRef& name,
+                            const llvm::Module &IRModule) {
+  if (llvm::Triple(IRModule.getTargetTriple()).isOSWindows()) {
+    // (SR-15938) Error when referencing #dsohandle in a Swift test on Windows
+    // On Windows, ignore the lack of __ImageBase in the TBD file.
```

Comment:
I would suggest that we add in the fact that this is a linker synthetic and therefore safe to ignore.

Reply:
Sure, will update the comment (but am going to let the current builds run to completion first.)